docs/source/guides/installation_guide.md

```diff
@@ -14,11 +14,16 @@
 * At least 1 GB of RAM is required for secure password-based key derivation with [scrypt](http://www.tarsnap.com/scrypt.html).
 * We have tested `nucypher` with Windows, Mac OS, and GNU/Linux (GNU/Linux is recommended).
 * If you don’t already have it, install [Python](https://www.python.org/downloads/). As of November 2019, we are working with Python 3.6, 3.7, and 3.8.
-* We also require the following system packages (Linux):
 
-    - `libffi-dev`
-    - `python3-dev`
-    - `python3-virtualenv`
+Before installing ``nucypher``, you may need to install necessary developer
+tools and headers, if you don't have them already. In Ubuntu, Debian, Linux Mint
+or similar distros, that is:
```

Comment:
To make it simple, we can recommend `sudo apt install python3-dev build-essential libffi-dev python3-pip` as one command